src/cli/install.rs

```diff
@@ -198,27 +198,20 @@ fn _run(cmd: &Command) -> anyhow::Result<()> {
             .with_context(|| format!("failed to write env file {:?}", settings.env_file))?;
     }
 
+    // We no longer support migration from the old layout.
     let base = home_dir()?.join(".edgedb");
     let new_layout = if base.exists() {
         eprintln!(
             "\
                 {BRANDING_CLI} no longer uses '{}' to store data \
                 and now uses standard locations of your OS. \
+                If you have migrated, you can safely ignore this message \
```

Comment:
I think it's not completely obvious that "migrated" here means "have run the `gel cli migrate` command in the past". Maybe "If you have used the CLI to update to the newer directory layout in the past..." or something like that?

Reply:
I could potentially just remove this check as I think it's pretty unlikely anyone will end up in this situation.